cmake/modules/hunter_download.cmake

```diff
@@ -493,6 +498,7 @@ function(hunter_download)
         "Configuration types: ${HUNTER_PACKAGE_CONFIGURATION_TYPES}"
     )
   endif()
+  hunter_status_debug("TLS_VERIFY: ${HUNTER_TLS_VERIFY}")
```

Comment:
I think it's better to use `HUNTER_TLS_VERIFY: ${HUNTER_TLS_VERIFY}` since this variable is documented.